src/index.ts

```diff
@@ -0,0 +1,97 @@
+import express, { Request, Response } from 'express';
+import swaggerJsdoc, { OAS3Definition } from 'swagger-jsdoc';
+import swaggerUi from 'swagger-ui-express';
+
+import { Application } from 'express';
+
+interface ServerSatatus {
+  status: string;
+  message: string;
+}
```

Comment:
It is more convenient to have interfaces and types in a separate file. As long as the project is small, you can start with the types.ts file, and then expand or divide it as needed.

Reply:
done